src/main/scala/strawman/collection/SortedMap.scala

```diff
@@ -26,6 +26,12 @@ trait SortedMapOps[K, +V, +CC[X, Y] <: SortedMap[X, Y] with SortedMapOps[X, Y, C
   def flatMap[K2, V2](f: ((K, V)) => IterableOnce[(K2, V2)])(implicit ordering: Ordering[K2]): CC[K2, V2] =
     sortedMapFromIterable(View.FlatMap(coll, f))
 
+  def collect[K2, V2](pf: PartialFunction[(K, V), (K2, V2)])(implicit ordering: Ordering[K2]): CC[K2, V2] =
+    flatMap { (kv: (K, V)) =>
+      if (pf.isDefinedAt(kv)) View.Single(pf(kv))
+      else View.Empty
+    }
```

Comment:
Note that the type annotation `kv: (K, V)` is required with both dotty and scalac 2.13.